db/migrate/20150612104023_add_gov_response_at_to_petitions.rb

```diff
@@ -0,0 +1,5 @@
+class AddGovResponseAtToPetitions < ActiveRecord::Migration
+  def change
+    add_column :petitions, :gov_response_at, :timestamp
```

Comment:
It might need to be ``parliament_response_at`` because it's not just the government that would be involved, right?  @yahoopete ?